MetaMask/metamask-mobile
# feat: 2795 disconnect all account permission multichain edit account screen

## **Description**
Behind feature flag for now, adds the ability to disconnect all accounts from the edit account screen.


## **Related issues**

Contributes to resolving: https://github.com/MetaMask/MetaMask-planning/issues/2795

## **Manual testing steps**

1. Connect one or more account(s) to a dapp in the in app browser
2. Select the dapp, in the header click the account avatar, a bottom sheet comes up, click Disconnect, a confirmation modal will show up.
3. Click Disconnect, then a toast should show up.

## **Screenshots/Recordings**


| Before       | After        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/89409b21-b2d1-4a6b-946d-af82f5fbb242"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/b3b9b1c0-2c88-467d-9b22-a9be239297b7"> |


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
